mne/time_frequency/spectrum.py

```diff
@@ -0,0 +1,1166 @@
+# -*- coding: utf-8 -*-
+"""Container classes for spectral data."""
+
+# Authors: Dan McCloy <dan@mccloy.info>
+#
+# License: BSD-3-Clause
+
+from copy import deepcopy
+from functools import partial
+from inspect import signature
+
+import numpy as np
+
+from ..channels.channels import UpdateChannelsMixin, _get_ch_type
+from ..channels.layout import _merge_ch_data
+# from ..defaults import (_BORDER_DEFAULT, _EXTRAPOLATE_DEFAULT,
+#                         _INTERPOLATION_DEFAULT)
```

Comment:
These commented-out things need to be addressed somehow sometime.

Reply:
these will be uncommented in a subsequent PR, where ALL the plot_topomap methods (for Cov, Proj, etc) will get their APIs standardized.